config/local_info.xml

```diff
@@ -179,6 +179,37 @@
     -->
     <restrict_personal_data>false</restrict_personal_data>
 
+    <!-- identity_providers
+      Shib/CheckIn Token reads config file
+      for needed AAI entitlements.
+      This is to allow Admins so that they will be empowered
+      to make quicker changes to AAI references.
+    -->
+    <identity_providers>
+      <provider>
+        <idp>aai.egi.eu/auth/realms/egi</idp>
+        <name>EGI Proxy</name>
+        <authentication_realms>
+          <shib_realm_name>EGI Proxy IdP</shib_realm_name>
+        </authentication_realms>
+        <required_groups>
+          <group>urn:mace:egi.eu:res:gocdb#aai.egi.eu</group>
+        </required_groups>
+        <help_url>https://docs.egi.eu/internal/configuration-database/access/#using-institutional-account-via-egi-check-in</help_url>
+      </provider>
+      <provider>
+        <idp>aai-demo.egi.eu/auth/realms/egi</idp>
+        <name>EGI Demo Proxy</name>
+        <authentication_realms>
+          <shib_realm_name>EGI Proxy IdP</shib_realm_name>
+        </authentication_realms>
```

Comment:
As with https://github.com/GOCDB/gocdb/pull/524/files#r1842006878

```suggestion
        <authentication_realm>
          EGI Proxy IdP
        </authentication_realm>
```

Reply:
Addressed as part of 01f3cd379b62c4b3119a25d09f7cef0d852b9904